package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "one-login-ga4",
+  "version": "1.0.0",
+  "description": "Reusable GA4 package for GDS One Login",
+  "main": "index.js",
+  "scripts": {
```

Comment:
Do we want to use minify? With the goal to reduce our npm package. 

I'm thinking about the build command also to add also. Maybe it's too early.
But if yes we want to use minify, we can start with a command like "npm run minify"

Reply:
Will add as part of a separate ticket: https://govukverify.atlassian.net/browse/DFC-38